framework_tvb/tvb/core/services/project_service.py

```diff
@@ -720,33 +716,43 @@ def _edit_data(self, datatype, new_data, from_group=False):
             operation = dao.get_operation_by_id(datatype.fk_from_operation)
             operation.user_group = new_group_name
             dao.store_entity(operation)
+            op_folder = self.structure_helper.get_project_folder(operation.project, str(operation.id))
+            vm_gid = json.loads(operation.parameters)['gid']
+            view_model_file = h5.determine_filepath(vm_gid, op_folder)
+            if view_model_file:
+                view_model_class = H5File.determine_type(view_model_file)
+                view_model = view_model_class()
+                with ViewModelH5(view_model_file, view_model) as f:
```

Comment:
@popaula937 this is one example of usage where fixing the previously mentioned TODO would help